src/App.js

```diff
@@ -64,6 +69,11 @@ function Alert(props) {
 	return <MuiAlert elevation={6} variant="filled" {...props} />
 }
 
+// set to the available wallet types
+let WalletType = null
+const { REACT_APP_INFURA_ID } = process.env
+if (!REACT_APP_INFURA_ID) throw new Error("Invalid Infura id")
```

Comment:
can we make this a non-fatal error and just disable walletconnect?
since otherwise if we forget to set it, the app is unusable